servers/src/main/java/tachyon/master/file/journal/SetTTLEntry.java

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
In the least, I would add a TODO to JournalEntry to figure out how to consolidate all the entries that modify the file / directory state (e.g. TTL / pinned / persisted / last modification time).